src/main/java/org/openhab/binding/zigbee/converter/ZigBeeConverterColorColor.java

```diff
@@ -135,43 +247,167 @@ public void run() {
 
     @Override
     public Channel getChannel(ThingUID thingUID, ZigBeeEndpoint endpoint) {
-        if (endpoint.getInputCluster(ZclColorControlCluster.CLUSTER_ID) == null
+        clusterColorControl = (ZclColorControlCluster) endpoint.getInputCluster(ZclColorControlCluster.CLUSTER_ID);
+
+        if (clusterColorControl == null || endpoint.getInputCluster(ZclOnOffCluster.CLUSTER_ID) == null
                 || endpoint.getInputCluster(ZclLevelControlCluster.CLUSTER_ID) == null) {
             return null;
         }
+
+        try {
+            if(!clusterColorControl.discoverAttributes(false).get()) {
+                logger.warn("{}: Cannot determine whether device endpoint supports RGB color. Assuming it does by now (checking again later)", endpoint.getEndpointAddress());
```

Comment:
This probably should be a debug level.

Reply:
Same as above. Here it is the discovery itself failing, so we are assuming the endpoint supports RGB color when it may not. In my understanding discovery should not normally fail, so hence the "warn", but I can put it as debug.